data/api/client-server/event_context.yaml

```diff
@@ -57,7 +57,10 @@ paths:
           type: integer
           name: limit
           description: |-
-            The maximum number of events to return. Default: 10.
+            The maximum number of context events to return. The limit applies
+            to the sum of the `events_before` and `events_after` arrays. The
```

Comment:
is there something to prove that this is true? It's generally sensible, though we do need evidence that this PR is a bug fix rather than a change.

Reply:
It's what synapse does: https://github.com/matrix-org/synapse/blob/v1.72.0/synapse/handlers/room.py#L1365-L1366

Let's just get this shipped.